.release/ci.hcl

```diff
@@ -0,0 +1,120 @@
+schema = "1"
+
+project "vault-lambda-extension" {
+  team = "vault"
+  slack {
+    notification_channel = "CRF6FFKEW"
+  }
+  github {
+    organization = "hashicorp"
+    repository = "vault-lambda-extension"
+    release_branches = ["main", "VAULT-5260/use-crt"]
```

Comment:
Leaving a comment as a reminder we should remove the test branch before merging.